test/e2e/data/infrastructure-vsphere/cluster-template.yaml

```diff
@@ -207,3 +209,22 @@ spec:
         kind: VSphereMachineTemplate
         name: '${ CLUSTER_NAME }'
       version: '${ KUBERNETES_VERSION }'
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "cni-${ CLUSTER_NAME }-crs-0"
+data: ${CNI_RESOURCES}
```

Comment:
I guess we should also export `CNI_RESOURCES` in `./hack/e2e.sh` ?

Reply:
Export happens in clusterctl framework `clusterctl.SetCNIEnvVar()`.